std_msgs/Header header
int32 robot_id  # LEFT=0, RIGHT=1
geometry_msgs/Pose pose  # Cartesian pose (position in mm, orientation in radians)
